packages/bundler-sdk-viem/src/BundlerAction.ts

```diff
@@ -1011,9 +998,15 @@ export namespace BundlerAction {
     signature: Hex,
     skipRevert = true,
   ): BundlerCall[] {
-    const { morpho } = getChainAddresses(chainId);
+    const {
+      morpho,
+      bundler3: { bundler3 },
+    } = getChainAddresses(chainId);
     const { r, s, yParity } = parseSignature(signature);
 
+    if (authorization.authorized === bundler3)
+      throw new BundlerErrors.UnexpectedSignature(authorization.authorized);
+
```

Comment:
Shuoldn't we do the same with permits and approvals?

Reply:
In these cases the space of spenders is very limited